client/src/utils/attributeAndParamUtils.js

```diff
@@ -426,22 +481,42 @@ const upsert = async () => {
     },
   });
 
-  const attributes = sessionStorage.getItem(APPLICATION_TASK_STORAGE_PATH);
+  const attributes = JSON.parse(
+    sessionStorage.getItem(APPLICATION_TASK_STORAGE_PATH)
+  );
+  let stillUsedAttributes = attributes.filter((singleAttribute) =>
+    usedIds.includes(singleAttribute.activityId)
+  );
```

Comment:
Just to check my understanding: Attributes, that are not used anymore but the ssot for that attribute is still existing, won't be deleted here? As far as I understood, only the attributes and params of deleted Bots will be deleted, isn't it that way?